lib/ui/text.dart

```diff
@@ -2278,6 +2352,11 @@ class StrutStyle {
   /// * `leading`: The minimum amount of leading between lines as a multiple of
```

Comment:
Probably want to improve documentation on this property as well to better distinguish it, since there are now two similarly named properties right next to each other.